dev-python/pypam/pypam-0.5.0-r7.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
Did you mean to include this in this PR? :)

Reply:
Oh, I see you did! By the way, I did notice you fixed the || die before, I just forgot/lost the open tab

Reply:
This was needed to add python 3.10 to xen-tools.